### ISSUE TITLE
Nestable lists (like workflowy)

### ISSUE THREAD

**Issue by DanielHeath** [schollz/cowyo#100]
I'm a big fan of 'workflowy' but it occurred to me that it's basically just structured lists, which cowyo already supports.

The syntax would be something like:
```
New Feature
 * it's helpful
 * can have lots of nesting
   * this one has sub-tasks
     * this is a sub-task
   * things to do
Other toplevel item
```

![screen shot 2018-01-03 at 10 51 00 pm](https://user-images.githubusercontent.com/47430/34519542-a197650e-f0d8-11e7-937a-4a76dc551386.png)

If I add this to my fork, would a PR be helpful?

**Comment by schollz** [comment]
Yes this would be great!

**Comment by chmac** [comment]
@DanielHeath Thanks for the tip on Workflowy, it looks like an awesome tool. I'd be even more excited to self host my own version, and Cowyo looks like a nice start. Did you make any progress with this idea?

**Comment by DanielHeath** [comment]
No, I'm unlikely to get to it for quite some time unfortunately.

**Comment by adimasduvitra** [comment]
Hi, I ran into a problem that nested list doesn't rendered properly on Chrome and other browsers. I'm using pure markdown syntax for this, or did I do something wrong?
